fix: tfrs-2050 - autocompletion

tim738745 opened the pull request:
@AlexZorkin I noticed that the problem seemed to be that a new component from react-search-autocomplete was used to replace the original component from react-autocomplete, but it looks like the necessary changes to complete this switch were not made; I think if we do want to switch to using react-search-autocomplete, that should be another card? For now I just reverted back to using react-autocomplete, so that the feature works.

AlexZorkin commented:
Gotcha, thanks Tim. Yes this was a missed fix during the code refactoring. If react-autocomplete works with the latest dependancies then we can just use that. Can you remove reference to react-search-autocomplete please.

tim738745 commented:
@AlexZorkin Found that react-search-autocomplete also replaced "react-bootstrap-autosuggest", but we can't go back to using that since react 15 is one of its peer dependencies; I'm going to try to use react-autocomplete to replace it instead.

tim738745 commented:
@AlexZorkin okay I removed react-search-autocomplete and replaced usage of react-bootstrap-autosuggest with our custom AutocompletedInput component (which uses react-autocomplete); should be okay now.
